Add integration test

shimos opened the pull request:
This PR introduces basic integration tests based on Hyperledger Fabric network created with the "test-network" script and "fabcar" chaincode in the fabric-samples repository.

shimos committed:
integration: add scripts to create network

This patch adds several scripts to create and destroy Hyperledger Fabric
network using "fabric-samples", which would be useful for integration
tests.

Signed-off-by: Taku Shimosawa <taku.shimosawa.uf@hitachi.com>

shimos committed:
Add separate jest config file

This patch separates configuration for jest from package.json, and adds
a dedicated config file (jest.unit.js).

Signed-off-by: Taku Shimosawa <taku.shimosawa.uf@hitachi.com>

shimos committed:
Add integration tests

This patch adds basic integration tests with Hyperledger Fabric v2.3.2,
using both the fabric-block and fabric-query2 network plugins.

The tests can be executed by "npm run integration-test".

Signed-off-by: Taku Shimosawa <taku.shimosawa.uf@hitachi.com>

shimos committed:
integration: Add 2.2.x test

This patch adds integration tests with Hyperledger Fabric v2.2.3.

Signed-off-by: Taku Shimosawa <taku.shimosawa.uf@hitachi.com>

satota2 reviewed (changes requested):
I think it would be better to add the copyright to the header of each file.

shimos committed:
integration: add license

This patch adds license and copyright lines to the beginning of the
script files.

Signed-off-by: Taku Shimosawa <taku.shimosawa.uf@hitachi.com>

shimos commented:
@satota2 Thanks. Added a commit to add license and copyright lines.

satota2 reviewed (approved):
LGTM!
